python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -66,7 +66,8 @@ def parse_node_data(in_file, feat_ops, node_id_col, label_conf, read_file):
         label_data = process_labels(data, label_conf, True)
         for key, val in label_data.items():
             feat_data[key] = val
-    return (data[node_id_col], feat_data)
+    node_ids = data[node_id_col] if node_id_col in data else None
```

Comment:
Why node_id_col does not exist in data?

Reply:
This is to handle the case that multiple node data blobs are provided for one node type. We only need to specify the node ID column in one of the blobs.